docs/error-reporting.md

```diff
@@ -0,0 +1,30 @@
+# Error Reporting Explained
+
+## What's going on?
+
+The Lighthouse team is constantly trying to improve the reliability of our tools. We've added functionality to anonymously report runtime exceptions, given your consent, using [Sentry](https://sentry.io/welcome/). We will use this information to detect new bugs and avoid regressions.
+
+Only CLI users are currently impacted. Node module consumers are opted out by default without a prompt, and the DevTools and extension integrations have no automatic error reporting functionality.
```

Comment:
maybe just include Node module consumers with Devtools and extension users for now since it won't be happening for them?

Reply:
yeah was trying to be explicit that the code isn't even there, but that was probably confusing